superduperdb/container/job.py

```diff
@@ -128,7 +128,7 @@ def run_on_dask(self, client, dependencies=()):
         :param client: dask client
         :param dependencies: list of dependencies
         """
-        CFG = s.configs.build_config()  # Why not use s.CFG or s.CFG.deepcopy()
+        CFG = s.configs.build_config()
```

Comment:
That comment was there because there's no reason to do this.

Reply:
That's because it's possible in the meantime that `env` variables have been set.

Reply:
@thejumpman2323 can you explain?

Reply:
Yes, so my rationale was that if the user creates a config during the setup and changes after some time, I was thinking to build it again if the change comes from env